powermon/commands/result.py

```diff
@@ -43,12 +43,12 @@ def set_device_id(self, device_id):
 
     def get_device_id(self) -> str:
         return self.device_id
-
-    def get_responses(self) -> list[Response]:
-        return self.responses
-
-    def add_responses(self, responses: list[Response]) -> bool:
-        self.responses.extend(responses)
+    
+    def get_responses(self) -> list[Reading]:
+        return self.readings
+    
+    def add_responses(self, responses: list[Reading]) -> bool:
```

Comment:
i guess these 'responses' should be called 'readings' too?

Reply:
yes, I've missed it in a few spots. Working on cleaning it up.